src/Router.ts

```diff
@@ -439,8 +439,19 @@ export class Router<C extends Context> extends Evented implements RouterInterfac
 		return this._outletContextMap.has(outletId);
 	}
 
-	getOutlet(outletId: string): OutletContext | undefined {
-		return this._outletContextMap.get(outletId);
+	getOutlet(outletId: string | Array<String>): OutletContext | undefined {
+		if (Array.isArray(outletId)) {
+			const outletContextArray = arrayFrom(this._outletContextMap);
+			const outlets = arrayFrom(outletId);
+
+			const matchingOutlet = find(outletContextArray, ([key]) => {
+				return includes(outlets, key);
+			});
+
+			return matchingOutlet ? matchingOutlet[1] : undefined;
+		} else {
```

Comment:
Don't need the else block as we are returning from within the if block

Reply:
Done. Would be really good if we linted for this, something like https://eslint.org/docs/rules/no-else-return for TS?

Reply:
It doesn't appear that tslint has it as an out of the box rule (https://palantir.github.io/tslint/rules/).  We haven't experimented with custom rules, but it maybe again worth opening an issue for tracking.  We looked at ESLints support of TypeScript about 9 months ago, and haven't since.  We know there has been work on settling on an AST syntax for TypeScript (preliminary work is already in Babel) but haven't had the time to see if ESLint has matured in that aspect.  We really need something that takes the AST from the TypeScript compiler.  Either way, it is likely a good idea for a separate issue to further discuss trying to move to ESLint.  I can see the long term benefits TBH.

Reply:
[Issue raised!](https://github.com/dojo/dojo2-package-template/issues/96)